.github/workflows/deploy.yml

```
@@ -4,10 +4,13 @@ on:
   push:
     branches:
     - master
+  pull_request:
+    branches:
+    - master
```

```suggestion
  pull_request: {}
```

PRs to any branch should run CI; the only thing that _shouldn't_ run is the deploy step